lib/server/index.js

```diff
@@ -74,6 +82,18 @@ app.use(async (req, res, next) => {
     }
   ];
 
+  _forEach(metaSettings, ( (value, key) => {
+    res.locals.header.push(
```

Comment:
* Check if setting has value before adding metatag, avoiding empty meta tags
* For coherence change `_forEach` to `forEach`:
* You can also remove the parentheses surrounding the cb function

```js
  forEach(metaSettings, (value, key) => {
    if (settings[key]) {
      res.locals.header.push({
        tag: 'meta',
        props:{
          name: value,
          content: safeHtmlString(settings[key])
        }
      });
    }
  });
```